cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -473,12 +474,14 @@ data ProtocolParametersUpdate =
        --
        protocolUpdateTreasuryCut :: Maybe Rational,
 
-       -- Introduced in Alonzo
+       -- Introduced in Babbage,
 
-       -- | Cost in ada per word of UTxO storage.
+       -- | Cost in ada per byte of UTxO storage.
        --
-       -- /Introduced in Alonzo/
-       protocolUpdateUTxOCostPerWord :: Maybe Lovelace,
```

Comment:
I think the proper way to do this would not be to modify existing Alonzo fields, but to add a new `protocolUpdateUTxOCostPerByte` field

Reply:
See my [comment](https://github.com/input-output-hk/cardano-node/pull/4141#discussion_r915908627), maybe we can get away with renaming. After Babbage is released creating PPUpdates for Alonzo doesn't make much sense, or does it?

In retrospective we should not have renamed it ledger. In particular using the binary flag `17` for perWord and perByte  was a mistake, IMHO. But that ship have sailed.

Reply:
It might for chains other than `mainnet` if we care about those?